src/options.js

```diff
@@ -3,7 +3,7 @@ const mcData = require('minecraft-data')
 // Minimum supported version (< will be kicked)
 const MIN_VERSION = '1.16.201'
 // Currently supported verson. Note, clients with newer versions can still connect as long as data is in minecraft-data
-const CURRENT_VERSION = '1.19.40'
+const CURRENT_VERSION = '1.19.41'
```

Comment:
This version refers to what path from minecraft-data should be used. 1.19.40 is correct here if no data changed between 1.19.40 and 1.19.41 (at which point a path should be in https://github.com/PrismarineJS/minecraft-data/blob/master/data/dataPaths.json )